src/Vlingo.Actors/Plugin/Logging/Console/ConsoleLogger.cs

```diff
@@ -34,16 +35,53 @@ public void Close()
         {
         }
 
-        public void Log(string message)
+        public void Debug(string message) => Log(message, "Debug");
+
+        public void Debug(string message, params object[] args) => Log(message, "Debug", args);
+
+        public void Debug(string message, Exception throwable) => Log(message, "Debug", throwable);
+
+        public void Error(string message) => Log(message, "Error");
+
+        public void Error(string message, params object[] args) => Log(message, "Error", args);
+
+        public void Error(string message, Exception throwable) => Log(message, "Error", throwable);
+
+        public void Info(string message) => Log(message, "Info");
+
+        public void Info(string message, params object[] args) => Log(message, "Info", args);
+
+        public void Info(string message, Exception throwable) => Log(message, "Info", throwable);
+
+        public void Trace(string message) => Log(message, "Trace");
+
+        public void Trace(string message, params object[] args) => Log(message, "Trace", args);
+
+        public void Trace(string message, Exception throwable) => Log(message, "Trace", throwable);
+
+        public void Warn(string message) => Log(message, "Warn");
+
+        public void Warn(string message, params object[] args) => Log(message, "Warn", args);
+
+        public void Warn(string message, Exception throwable) => Log(message, "Warn", throwable);
+
+        private void Log(string message, string level)
+        {
+            System.Console.WriteLine($"{Name}[{level}]: {message}");
+        }
+
+        private void Log(string message, string level, Exception ex)
         {
-            System.Console.WriteLine($"{Name}: {message}");
+            System.Console.WriteLine($"{Name}[{level}]: {message}");
+            System.Console.WriteLine($"{Name}[{level}] [Exception]: {ex.Message}");
```

Comment:
I think that here we should inspect for `InnerException` property in a case we receive `AggregateException` or someone will provide the inner exception.

Reply:
Good point. Addressing it...